src/battle_main.c

```diff
@@ -4984,7 +4984,14 @@ void SetTypeBeforeUsingMove(u16 move, u8 battlerAtk)
     }
     else if (gBattleMoves[move].effect == EFFECT_JUDGMENT)
     {
-        // TODO:
+        #ifdef IS_ITEM_PLATE
+            if (!(IS_ITEM_PLATE(gBattleMons[battlerAtk].item)))
+                gBattleStruct->dynamicMoveType = TYPE_NORMAL;
```

Comment:
not necessary explicitly set to normal type: if you don't set `dynamicMoveType`, it will use the type defined with the const move data (which is Normal in this case)